### PR TITLE
fix: Update Kubernetes cluster example

### PR DESCRIPTION
**Pull request by tkyi**
Updating the Kubernetes cluster example to use PostgreSQL database instead of DynamoDB since DynamoDB integration is now deprecated in Screwdriver.

### CODE REVIEW

File: docs/cluster-management/kubernetes.md [hunk 1]
```diff
@@ -12,28 +12,29 @@ A Screwdriver cluster consists of a Kubernetes cluster running the Screwdriver A
...
 
 ## Create your Kubernetes cluster
-Follow instructions at [Getting Started with AWS](http://kubernetes.io/docs/getting-started-guides/aws/).
+Follow instructions at [Running Kubernetes on AWS EC2](http://kubernetes.io/docs/getting-started-guides/aws/).
 
 
-## Setup Kubernetes secrets
-A [Secret](http://kubernetes.io/docs/user-guide/secrets/) is an object that contains a small amount of sensitive data such as a password, a token, or a key. You will need to setup secrets for Screwdriver for your cluster to work properly.
+## Setup Screwdriver secrets
+After creating your Kubernetes cluster, you'll need to populate it with some secrets that will give you access to to your database and Github.
```
Comment (r3rastogi) [inline docs/cluster-management/kubernetes.md]: "access to to" :(

File: docs/cluster-management/kubernetes.md [hunk 2]
```diff
@@ -12,28 +12,29 @@ A Screwdriver cluster consists of a Kubernetes cluster running the Screwdriver A
...
+A [Secret](http://kubernetes.io/docs/user-guide/secrets/) is an object that contains a small amount of sensitive data such as a password, a token, or a key.
```
Comment (r3rastogi) [inline docs/cluster-management/kubernetes.md]: "such as a password, token, or key"

File: docs/cluster-management/kubernetes.md [hunk 3]
```diff
@@ -12,28 +12,29 @@ A Screwdriver cluster consists of a Kubernetes cluster running the Screwdriver A
...
+| :-------------------- | :-------------|
+| SECRET_JWT_PRIVATE_KEY | A private key used for signing JWT tokens |
+| SECRET_JWT_PUBLIC_KEY | A public key used for signing JWT tokens |
+| DATASTORE_SEQUELIZE_DATABASE | Sequelize database name |
+| DATASTORE_SEQUELIZE_USERNAME | Sequelize database username |
+| DATASTORE_SEQUELIZE_PASSWORD | Sequelize database password |
+| SECRET_OAUTH_CLIENT_ID | The client ID used for [OAuth](https://developer.github.com/v3/oauth) with Github |
+| SECRET_OAUTH_CLIENT_SECRET | The client secret used for OAuth with github |
+| WEBHOOK_GITHUB_SECRET | Secret to add to GitHub webhooks so that we can validate them |
+| SECRET_PASSWORD | A password used for encrypting session, and OAuth data. Can be anything. **Needs to be minimum 32 characters** |
```
Comment (r3rastogi) [inline docs/cluster-management/kubernetes.md]: "A password used for encrypting session and OAuth data; can be anything."

File: docs/cluster-management/kubernetes.md [hunk 4]
```diff
@@ -12,28 +12,29 @@ A Screwdriver cluster consists of a Kubernetes cluster running the Screwdriver A
...
-| SECRET_JWT_PRIVATE_KEY | Yes | A private key used for signing JWT tokens. |
-| SECRET_JWT_PUBLIC_KEY | Yes | A public key used for signing JWT tokens. |
-| WEBHOOK_GITHUB_SECRET | Yes | Secret to add to GitHub webhooks so that we can validate them |
-| SECRET_PASSWORD | Yes | A password used for encrypting session, and OAuth data. Can be anything. **Needs to be minimum 32 characters** |
-| K8S_TOKEN | Yes | Your Kubernetes <DEFAULT_TOKEN_NAME> |
+| Secret Key            | Description |
+| :-------------------- | :-------------|
+| SECRET_JWT_PRIVATE_KEY | A private key used for signing JWT tokens |
+| SECRET_JWT_PUBLIC_KEY | A public key used for signing JWT tokens |
+| DATASTORE_SEQUELIZE_DATABASE | Sequelize database name |
```
Comment (FenrirUnbound) [inline docs/cluster-management/kubernetes.md]: Might be better to use a less specific word that "Sequelize", like "SQL"

File: docs/cluster-management/kubernetes.md [hunk 5]
```diff
@@ -90,8 +106,9 @@ metadata:
 type: Opaque
 data:
   # make sure the values are all base64 encoded
-  dynamodbid: someid
-  dynamodbsecret: somesecret
+  dbhost: dbhostnamehere
+  dbusername: mydatabase
+  dbpassword: somepassword
   password: MWYyZDFlMmU2N2Rm
```
Comment (r3rastogi) [inline docs/cluster-management/kubernetes.md]: lol what is this password

File: docs/cluster-management/kubernetes.md (hunk 4 above)
Comment (tkyi) [reply docs/cluster-management/kubernetes.md]: Is that less specific? I'll fix it 👍

File: docs/cluster-management/kubernetes.md (hunk 5 above)
Comment (tkyi) [reply docs/cluster-management/kubernetes.md]: I put the base64 encoded one
Should I just make it consistent?

Comment (r3rastogi) [reply docs/cluster-management/kubernetes.md]: nah it's fine; it makes sense if you read the whole section 👍

**Review by r3rastogi — APPROVED** [review]

**Comment by r3rastogi** [comment]
bitmoji

**Pull request merged**